config_test.go

```diff
@@ -119,6 +119,30 @@ func TestMerge_HttpsOptions(t *testing.T) {
 	}
 }
 
+func TestMerge_BasicAuthOptions(t *testing.T) {
+	{
```

Comment:
Nitpick: We don't need to wrap this test in a block since the variables aren't used in any other scope.